src/components/TokenBalance.test.tsx

```diff
@@ -120,7 +121,17 @@ describe('FiatExchangeTokenBalance and HomeTokenBalance', () => {
     async (TokenBalanceComponent) => {
       const store = createMockStore({
         tokens: {
-          tokenBalances: {},
+          tokenBalances: {
```

Comment:
just checking, without this change does this test still pass? for a new install, the initial value for `tokenBalances` is `{}` so would be nice to test this base case

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/2645/commits/653519e04f1d418a0b98c79679747670e0e89868!